src/render/mod.rs

```diff
@@ -161,6 +161,20 @@ impl<'world, 'state, 'world2, 'state2, 'a, 'b, 'c, 'v, 'p>
                     };
                 collider_debug == ColliderDebug::AlwaysRender
             }
+            DebugRenderObject::RigidBody(h, ..) => {
+                let Some(collider) = self.context.bodies.get(h) else {
+                    return false;
+                };
+                let entity = Entity::from_bits(collider.user_data as u64);
+
+                let collider_debug =
+                    if let Ok(collider_override) = self.override_visibility.get(entity) {
+                        *collider_override
+                    } else {
+                        self.default_collider_debug
+                    };
+                collider_debug == ColliderDebug::AlwaysRender
+            }
```

Comment:
This is outside of this PR's scope, but helps with suppressing debug drawing for rigidbodies ; we could parameterize more the `ColliderDebug` but I'm leaving it as a future exercise on user feedback for now.